packages/actor-init-sparql/lib/HttpServiceSparqlEndpoint.ts

```diff
@@ -25,18 +25,21 @@ export class HttpServiceSparqlEndpoint {
 
   public static readonly HELP_MESSAGE = `comunica-sparql-http exposes a Comunica engine as SPARQL endpoint
 
-context should be a JSON object or the path to such a JSON file.
-
 Usage:
-  comunica-sparql-http context.json [-p port] [-t timeout] [-l log-level] [-i] [--help]
-  comunica-sparql-http "{ \\"sources\\": [{ \\"type\\": \\"hypermedia\\", \\"value\\" : \\"http://fragments.dbpedia.org/2015/en\\" }]}" [-p port] [-t timeout] [-l log-level] [-i] [--help]
+  comunica-sparql-http -c context.json [-p port] [-t timeout] [-l log-level] [-i] [-b] [--help] [--version] [--lenient]
+  comunica-sparql-http -c "{ \\"sources\\": [{ \\"type\\": \\"hypermedia\\", \\"value\\" : \\"http://fragments.dbpedia.org/2015/en\\" }]}" [-p port] [-t timeout] [-l log-level] [-i] [-b] [--help] [--version] [--lenient]
+  comunica-sparql-http hypermedia@http://fragments.dbpedia.org/2015/en [-p port] [-t timeout] [-l log-level] [-i] [-b] [--help] [--version] [--lenient]
```

Comment:
I would move this line to be the first example (so it's clear that this is the "proper" way of calling it).
I would also add an example with 2 sources (like the other help message).